packages/wonder-blocks-testing/src/fixtures/types.js

```diff
@@ -0,0 +1,180 @@
+// @flow
+import * as React from "react";
+import type {StorybookOptions} from "./adapters/storybook.js";
+
+/**
+ * Options injected to the function that returns the fixture props.
+ */
+export type GetPropsOptions = {|
+    /**
+     * A function to call that will log output.
+     */
+    log: (message: string, ...args: Array<any>) => void,
+|};
+
+/**
+ * Adapter options keyed by the adapter name.
+ */
+export type AdapterOptions = {|
+    storybook?: StorybookOptions,
+    [adapterName: string]: {...},
+|};
+
+/**
+ * Options to describe a collection of fixtures.
+ */
+export type FixturesOptions<TProps: {...}> = {|
+    /**
+     * The component being tested by the fixtures.
+     */
+    component: React.ComponentType<TProps>,
+
+    /**
+     * Optional title of the fixture collection.
+     */
+    title?: string,
```

Comment:
Any plans to make this non-optional in the future?

Reply:
No - the reason it's optional is that many folks will just want this to be the component name, which is standard for storybook - so providing no title defaults to that.